source/NVDAObjects/__init__.py

```diff
@@ -979,6 +980,8 @@ def event_becomeNavigatorObject(self, isFocus=False):
 		if braille.handler.shouldAutoTether and isFocus:
 			return

```

Comment:
@josephsl: Could you please look at this early return? It looks like your code is going to play object coordinates for focus events when auto tethering is off. You probably want to remove the early return and change it like this:

```
		if not isFocus:
			if not braille.handler.shouldAutoTether:
				braille.handler.handleReviewMove(shouldAutoTether=not isFocus)
			if config.conf["presentation"]["playObjectCoordinates"]:
				screenExplorer.playObjectCoordinates(self)
```